llvm/include/llvm/CodeGen/GlobalISel/MachineIRBuilder.h

```diff
@@ -1888,6 +1888,14 @@ class MachineIRBuilder {
     return buildInstr(TargetOpcode::G_FRINT, {Dst}, {Src0}, Flags);
   }
 
+  /// Build and insert \p Dst = G_INTRINSIC_ROUNDEVEN \p Src0, \p Src1
+  MachineInstrBuilder
+  buildintrinsicRoundeven(const DstOp &Dst, const SrcOp &Src0,
```

Comment:
Capitalization is off. I'd also probably just drop the intrinsic part from the name (we should also drop it from the opcode name)